Api/Validator.php

```diff
@@ -0,0 +1,153 @@
+<?php
+    
+    namespace Metrilo\Analytics\Api;
+    
+    class Validator {
+        private $logDestination = __DIR__ . '/MetriloApiErrors.log'; // construct log path in ajax controller and pass it as constructor param for Validator class
```

Comment:
This wouldn't be that simple. The platform (woo/m1/m2) specific code would have to pass some sort of Logger method to here, that could be different for each platform, but would have the same shared interface